src/main/java/com/revature/chronicle/TestController/HelloController.java

```diff
@@ -0,0 +1,18 @@
+package com.revature.chronicle.TestController;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("hello")
+public class HelloController {
+
+    @GetMapping(consumes = MediaType.APPLICATION_JSON_VALUE,
+            produces= MediaType.APPLICATION_JSON_VALUE)
+    public ResponseEntity<String> getHello() {
+        return new ResponseEntity<>("Hello World", HttpStatus.OK);
```

Comment:
@justinkroh22 this one could stay as long as you give it an `@Profile` and remove the `@RestController` in com.revature.chronicle.ChronicleApplication

Reply:
Removed all test controllers